### ISSUE TITLE
Animatable properties for particle systems

### ISSUE THREAD

**Issue by vlidholt** [spritebuilder/SpriteBuilder#24]
Make more properties on the particle systems animatable on the timeline, including opacity, color, size and speed.

**Comment by haligalaph** [comment]
It also would be great to be able to start/stop particle systems from the timeline.

**Comment by vlidholt** [comment]
Hi, you can already start/stop the particles by toggling the visibility.

**Comment by haligalaph** [comment]
Yes, it works fine for systems with long duration. But will it work for explosions-like systems? I mean, will it restart on changing visibility?

**Comment by vlidholt** [comment]
Yes, it will restart.

**Comment by haligalaph** [comment]
It is great, thank you.
